app/models/concerns/clone/report_template.rb

```diff
@@ -10,8 +10,7 @@ def initialize(source, **options)
     options.delete(:clean_formulas)
     options.delete(:hidden)
     options.delete(:position)
-    options.delete(:params)
-    @outcome.assign_attributes(options)
+    @outcome.assign_attributes(options[:params])
```

Comment:
Just wondering, is there a reason to delete these things from `options` when afterward we only assign `options[:params]` to the outcome anyway?

Reply:
Yes, it is possible to pass all these parameters depending on how you run this code (module Clone#clone)